### PR TITLE
Fix/sponsors not displayed

### PR DESCRIPTION
**Pull request by walerowicz-miam**

### CODE REVIEW

File: Sources/CoursesUxMiamFramework/ui/RecipeCard/CoursesUStandaloneRecipeCard.swift [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+                                    Image(packageResource: "MealIdeas", ofType: "png")
+                                        .resizable()
+                                        .frame(width: 119, height: 40)
+                                    Spacer()
+                                }
+                            }.padding(dimensions.mPadding)
+                            Spacer()
+                            HStack {
+                                /*if params.recipe.isSponsored{
+                                    if let urlString = params.recipe.relationships?.sponsors?.data.first?.attributes?.logoUrl, let url = URL(string: urlString) {
```
Comment (diarmuidmcg) [inline Sources/CoursesUxMiamFramework/ui/RecipeCard/CoursesUStandaloneRecipeCard.swift:62]: should this still be commented?

File: Sources/CoursesUxMiamFramework/ui/RecipeCard/CoursesUStandaloneRecipeCard.swift [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+                    HStack(spacing:0) {
+                        CoursesUPricePerPerson(pricePerGuest: params.recipe.attributes?.price?.pricePerServe ?? params.recipePrice)
+                        Spacer()
+
+                        CoursesULikeButton(recipeId: params.recipe.id)
+                        /*LikeButton(
+                            likeButtonInfo: LikeButtonInfo(
+                                recipeId: params.recipe.id,
+                                iconSize: CGSize(width: 20, height: 20),
+                                backgroundColor: Color.white
```
Comment (diarmuidmcg) [inline Sources/CoursesUxMiamFramework/ui/RecipeCard/CoursesUStandaloneRecipeCard.swift:100]: & here?

File: Sources/CoursesUxMiamFramework/ui/RecipeCard/CoursesUStandaloneRecipeCard.swift [hunk 3]
```diff
@@ -0,0 +1,163 @@
...
+                
+                
+            }
+        }
+    }
+}
+
+
+
+
```
Comment (diarmuidmcg) [inline Sources/CoursesUxMiamFramework/ui/RecipeCard/CoursesUStandaloneRecipeCard.swift:163]:
```suggestion
            }
        }
    }
}
```

**Review by diarmuidmcg — COMMENTED** [review]
just some questions on dead code

File: Sources/CoursesUxMiamFramework/ui/RecipeCard/CoursesUStandaloneRecipeCard.swift (hunk 1 above)
Comment (walerowicz-miam) [reply Sources/CoursesUxMiamFramework/ui/RecipeCard/CoursesUStandaloneRecipeCard.swift:62]: Yes, it's to be the same as android, until we can make a correction

File: Sources/CoursesUxMiamFramework/ui/RecipeCard/CoursesUStandaloneRecipeCard.swift (hunk 2 above)
Comment (walerowicz-miam) [reply Sources/CoursesUxMiamFramework/ui/RecipeCard/CoursesUStandaloneRecipeCard.swift]: done

File: Sources/CoursesUxMiamFramework/ui/RecipeCard/CoursesUStandaloneRecipeCard.swift (hunk 3 above)
Comment (walerowicz-miam) [reply Sources/CoursesUxMiamFramework/ui/RecipeCard/CoursesUStandaloneRecipeCard.swift]: done

**Review by diarmuidmcg — APPROVED** [review]

**Pull request merged**